docs/references/subcommands/container/scanner.md

```diff
@@ -213,6 +213,11 @@ It will then report them to FOSSA which will try to match the Jar files to the p
 This process relies on there being a back-end that can perform that analysis.
 SaaS customers should have this functionality available but on-prem customers may need to contact FOSSA support to have it enabled.
 
+### Distroless Containers
```

Comment:
The ToC didn't get updated with this new header. You need to add this between the "Container Jar Analysis"  and "Supported Container Package Managers" sections:

```
    - [Distroless Containers](#distroless-containers)
```

Reply:
Good catch, I've updated this.